### PR TITLE
[Feature] Torchvision high-precision ResNet model

### PR DESCRIPTION
**Pull request by jbwang1997**
## Motivation

Torchvision released its high-precision ResNet models. The training details can be found on the [Pytorch website](https://pytorch.org/blog/how-to-train-state-of-the-art-models-using-torchvision-latest-primitives/). Here, we have done grid searches on learning rate and weight decay and found the optimal hyper-parameter on the detection task.

### CODE REVIEW

File: configs/faster_rcnn/faster_rcnn_r50_fpn_tnr-pretrain_1x_coco.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+]
+
+checkpoint = 'https://download.pytorch.org/models/resnet50-11ad3fa6.pth'
+model = dict(
+    backbone=dict(init_cfg=dict(type='Pretrained', checkpoint=checkpoint)))
+
+optimizer = dict(
+    _delete_=True,
+    type='AdamW',
+    lr=0.0001,
```
Comment (ZwwWayne) [inline configs/faster_rcnn/faster_rcnn_r50_fpn_tnr-pretrain_1x_coco.py:14]: Add comments here to indicate which part is searched to be optimal.

Comment (jbwang1997) [reply configs/faster_rcnn/faster_rcnn_r50_fpn_tnr-pretrain_1x_coco.py:15]: Done.

File: configs/faster_rcnn/metafile.yml [hunk 2]
```diff
@@ -405,3 +405,23 @@ Models:
         Metrics:
           box AP: 43.1
     Weights: https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_x101_64x4d_fpn_mstrain_3x_coco/faster_rcnn_x101_64x4d_fpn_mstrain_3x_coco_20210524_124528-26c63de6.pth
+
+  - Name: faster_rcnn_r50_fpn_tnr-pretrain_1x_coco
+    In Collection: Faster R-CNN
+    Config: configs/faster_rcnn/faster_rcnn_r50_fpn_tnr-pretrain_1x_coco
```
Comment (RangiLyu) [inline configs/faster_rcnn/metafile.yml:411]:
```suggestion
    Config: configs/faster_rcnn/faster_rcnn_r50_fpn_tnr-pretrain_1x_coco.py
```

Comment (jbwang1997) [reply configs/faster_rcnn/metafile.yml]: Done.

**Review by ZwwWayne — APPROVED** [review]
